src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -454,9 +488,9 @@ public void Dispose()
             Dispose(true);
         }
 
-        private bool IsTriggerMetadataPopulatedByWorker()
```

Comment:
Why is this getting removed? maybe a merge conflict?

Reply:
oh no! yes. Thanks for catching that.